README.md

```diff
@@ -131,6 +97,18 @@ See the
 repository for Ansible scripts that build Docker images and deploy containers
 to the test and production environments.
 
+# Running Tests
+
+Tests require a locally-installed version of Chrome to support feature specs
+
+The usual ENV variables need to be set, for now
+
+- DL Chrome @ `https://commondatastorage.googleapis.com/chromium-browser-snapshots/index.html?prefix=Linux_x64/737173/`
+- Extract to `PATH_OF_YOUR_CHOOSING`
+- Precompile assets for `test` (why???): `RAILS_ENV=test bundle exec rake assets:precompile`
```

Comment:
I agree, that this shouldn't be necessary in the test environment. There might be something wrong with the test environment file. 
In development, this flag is set: 
https://github.com/upenn-libraries/discovery-app/blob/master/config/environments/development.rb#L28 and it should probably be set for the test environment as well.

Reply:
I'm not sure this is required anymore. I've cleared out `public/assets` and I'm not having any issues with the suite.

Reply:
I'm going to remove this step until someone else runs into an issue